packages/dds/tree/src/test/testTrees.ts

```diff
@@ -174,126 +163,118 @@ export const anyFields = builder.object("anyFields", {
-	testTree("true boolean", library, leaf.boolean, true),
-	testTree("false boolean", library, leaf.boolean, false),
-	testTree("hasMinimalValueField", library, hasMinimalValueField, {
-		[typedJsonCursor.type]: hasMinimalValueField,
-		field: { [typedJsonCursor.type]: minimal },
-	}),
-	testTree("hasNumericValueField", library, hasNumericValueField, {
-		[typedJsonCursor.type]: hasNumericValueField,
-		field: 5,
-	}),
-	testTree("hasPolymorphicValueField", library, hasPolymorphicValueField, {
-		[typedJsonCursor.type]: hasPolymorphicValueField,
-		field: 5,
-	}),
-	testTree("hasOptionalField-empty", library, hasOptionalField, {
-		[typedJsonCursor.type]: hasOptionalField,
-	}),
+	{
+		name: "node-with-identifier-field",
+		schemaData: toFlexSchema(HasIdentifierField),
+		treeFactory: (idCompressor?: IIdCompressor) => {
+			assert(idCompressor !== undefined, "idCompressor must be provided");
+			const id = idCompressor.decompress(idCompressor.generateCompressedId());
+			return [jsonableTreeFromCursor(cursorFromInsertable(HasIdentifierField, { field: id }))];
+		},
+		policy: defaultSchemaPolicy,
+	},
+	{
+		name: "identifier-field",
+		schemaData: toFlexSchema(factory.identifier),
+		treeFactory: (idCompressor?: IIdCompressor) => {
+			assert(idCompressor !== undefined, "idCompressor must be provided");
+			const id = idCompressor.decompress(idCompressor.generateCompressedId());
+			return [{ type: leaf.string.name, value: id }];
+		},
+		policy: defaultSchemaPolicy,
+	},
+	testSimpleTree("true boolean", factory.boolean, true),
+	testSimpleTree("false boolean", factory.boolean, false),
+	testSimpleTree("hasMinimalValueField", HasMinimalValueField, { field: {} }),
+	testSimpleTree("hasNumericValueField", HasNumericValueField, { field: 5 }),
+	testSimpleTree("hasPolymorphicValueField", HasPolymorphicValueField, { field: 5 }),
+	testSimpleTree("hasOptionalField-empty", HasOptionalField, {}),
 	testTree("allTheFields-minimal", library, allTheFields, {
-		[typedJsonCursor.type]: allTheFields,
-		valueField: 5,
+		type: allTheFields.name,
+		fields: { valueField: [{ type: leaf.number.name, value: 5 }] },
 	}),
 	testTree("allTheFields-full", library, allTheFields, {
-		[typedJsonCursor.type]: allTheFields,
-		valueField: 5,
-		optional: 5,
-		sequence: [5],
+		type: allTheFields.name,
+		fields: {
+			valueField: [{ type: leaf.number.name, value: 5 }],
+			optional: [{ type: leaf.number.name, value: 5 }],
+			sequence: [{ type: leaf.number.name, value: 5 }],
+		},
 	}),
 	testTree("anyFields-minimal", library, anyFields, {
-		[typedJsonCursor.type]: anyFields,
-		valueField: 5,
+		type: anyFields.name,
+		fields: { valueField: [{ type: leaf.number.name, value: 5 }] },
 	}),
 	testTree("anyFields-full", library, anyFields, {
-		[typedJsonCursor.type]: anyFields,
-		valueField: 5,
-		optional: 5,
-		sequence: [5, { [typedJsonCursor.type]: minimal }],
-	}),
-	testTree("escapedFields", library, escapedFieldProperties, {
```

Comment:
field escaping was removed recently, so this test tree case, and its related snapshot files are no longer needed, and were removed